candle-wasm-examples/segment-anything/src/bin/m.rs

```diff
@@ -74,17 +75,31 @@ impl Model {
         Ok(())
     }
 
-    // x and y have to be between 0 and 1
-    pub fn mask_for_point(&self, x: f64, y: f64) -> Result<JsValue, JsError> {
-        if !(0. ..=1.).contains(&x) {
-            Err(JsError::new(&format!(
-                "x has to be between 0 and 1, got {x}"
-            )))?
-        }
-        if !(0. ..=1.).contains(&y) {
-            Err(JsError::new(&format!(
-                "y has to be between 0 and 1, got {y}"
-            )))?
+    pub fn mask_for_point(&self, points: js_sys::Array) -> Result<JsValue, JsError> {
+        let transformed_points: Vec<(f64, f64, bool)> = points
+            .values()
+            .into_iter()
+            .map(|val| {
+                let point_array: js_sys::Array = val.unwrap().into();
+                let x: f64 = point_array.get(0).as_f64().unwrap();
+                let y: f64 = point_array.get(1).as_f64().unwrap();
+                (x, y, true)
+            })
+            .collect::<Vec<_>>();
+        
+        for &(x, y, bool) in &transformed_points {
```

Comment:
cargo clippy complained about the unused named `bool` here  suggests `bool` -> `_bool`
```rust
for &(x, y, _bool) in &transformed_points {
```